spec/takeaway_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:takeaway)    { Takeaway.new }
+  let(:food_item_1) { double(:food_item_1, :name => "Pizza", :price => 15.00) }
+  let(:food_item_2) { double(:food_item_2, :name => "Chips", :price => 5.00) }
+  let(:food_item_3) { double(:food_item_3, :name => "Sausages", :price => 7.00) }
+  # let(:menu)        { double(:menu, check_menu_for(food_item) => false) }
+
+  it 'can display the menu' do
+    takeaway.use_default_menu
```

Comment:
Could make use of a double here to decouple the dependency on takeaway from menu. It looks like this was your intention from your code design and the commented out line of code - did you have trouble implementing this?